### PR TITLE
feat(hadoop): Add hdfs-utils

### PR DESCRIPTION
**Pull request by sbernauer**
# Description

Depends on https://github.com/stackabletech/hdfs-group-mapper/pull/2

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
- [ ] Changes are OpenShift compatible
- [ ] All added packages (via microdnf or otherwise) have a comment on why they are added
- [ ] Things not downloaded from Red Hat repositories should be mirrored in the Stackable repository and downloaded from there
- [ ] All packages should have (if available) signatures/hashes verified
- [ ] Does your change affect an SBOM? Make sure to update all SBOMs
```

### CODE REVIEW

**Comment by lfrancke** [comment]
What are the hdfs-utils?

File: conf.py [hunk 1]
```diff
@@ -84,6 +84,7 @@
                 "async_profiler": "2.9",
                 "jmx_exporter": "0.20.0",
                 "protobuf": "2.5.0",
+                # no HDFS utils, as e.g. patches from https://github.com/apache/hadoop/pull/6553 are missing. 3.2.x will probably removed shortly anyway.
```
Comment (adwk67) [inline conf.py:87]:
```suggestion
                # no HDFS utils, as e.g. patches from https://github.com/apache/hadoop/pull/6553 are missing. 3.2.x will probably removed in version 24.3 anyway.
```

**Review by adwk67 — APPROVED** [review]
lgtm. Just one nit, which i'll leave up to you (so that the comment about the removal still makes sense in the distant future!)

File: conf.py (hunk 1 above)
Comment (sbernauer) [reply conf.py:87]: Let's hope for it :)

**Review by adwk67 — APPROVED** [review]

**Pull request merged**